geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistenceAdvisorImpl.java

```diff
@@ -1161,11 +1162,19 @@ public void checkInterruptedByShutdownAll() {
 
   @Override
   public void close() {
-    isClosed = true;
-    persistentMemberManager.removeRevocationListener(profileChangeListener);
-    cacheDistributionAdvisor.removeProfileChangeListener(profileChangeListener);
-    persistentStateListeners = Collections.emptySet();
-    releaseTieLock();
+    try {
+      synchronized (this) {
+        isClosed = true;
+        persistentMemberManager.removeRevocationListener(profileChangeListener);
+        cacheDistributionAdvisor.removeProfileChangeListener(profileChangeListener);
+        persistentStateListeners = Collections.emptySet();
+        releaseTieLock();
+      }
+    } catch (CancelException e) {
+      logger.debug("persistence advisor close abridged due to shutdown", e);
+    } catch (Exception ex) {
```

Comment:
If there is Exception other than `CancelException`, I don't think the Exception can be ignored. It should be thrown.

Reply:
as we are in the cleanup phase, if we throw the exception, the rest of the cleanup will be skipped, which is not what we want. This behavior is consistent with other advisor close method in the `DistributedRegion`, as in DLockService.close and `DistributionAdvisor.close`